HISTORY.md

```diff
@@ -0,0 +1,45 @@
+## v0.6 (2015-09-10)
```

Comment:
There were no tags for v0.5 and v0.6 and it wasn't going to be easy to add them.  See https://github.com/voxpupuli/hiera-eyaml-gpg/issues/32